README.md

```diff
@@ -18,3 +18,6 @@ Follow the Reader SDK [Quick Start Guide](https://docs.connect.squareup.com/paym
     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
     See the License for the specific language governing permissions and
     limitations under the License.
+
+## Feedback
+Leave a feedback for this sample app [here](https://delighted.com/t/DT6msOcY)!
```

Comment:
I find this a little confusing. We are asking to leave a feedback about our sample app and the actual feedback page is asking "How helpful is this page?"

Reply:
We plan to change this link soon so that we can get different links for different apps. 
In the new link, we will also change the wording of the question to be more generic.